cms/envs/dev_with_worker.py

```diff
@@ -3,29 +3,69 @@
 requirement of a celery worker running in the background to process
 celery tasks.
 
-The worker can be executed using:
+When testing locally, run lms/cms with this settings file as well, to test queueing
+of tasks onto the appropriate workers.
 
-django_admin.py celery worker
+In two separate processes on devstack:
+    paver devstack studio --settings=dev_with_worker
+    ./manage.py cms celery worker --settings=dev_with_worker
```

Comment:
A note: this is actually broken for me - something is wrong with how logging is configured with dev settings on my devstack.

Is there still a difference between `dev` and `devstack`? Should this file be moved to `devstack_with_workers`?